pages/linux/objdump.md

```diff
@@ -0,0 +1,15 @@
+# objdump
+
+> View information on object files.
+
+- Display the file header information:
+
+`objdump -f {{binary}}`
+
+- Display the dis-assembled output of executable sections:
+
+`objdump -d {{binary}}`
+
+- Display complete binary hex dump of all sections:
```

Comment:
"Display complete binary hex dump" -> "Display _a_ complete binary hex dump"?

Also, what's a section? Is it something that the average go / c++ would know about / understand?

Reply:
> Also, what's a section?

An ELF binary is comprised of several sections, which the OS knows how to read. To be specific its the format of the binary file.

Yes, I don't think this is too complicated. Atleast for the person going to run `objdump`.